packages/compat/package.json

```diff
@@ -13,16 +13,23 @@
   "bugs": {
     "url": "https://github.com/solarwindscloud/solarwinds-apm-js/issues"
   },
-  "main": "dist/index.js",
-  "types": "dist/index.d.ts",
+  "type": "module",
+  "exports": {
+    ".": {
+      "import": "./dist/es/index.js",
+      "require": "./dist/cjs/index.js"
+    }
+  },
+  "main": "./dist/cjs/index.js",
   "files": [
-    "dist/"
+    "./src/",
+    "./dist/"
   ],
```

Comment:
Including `src` in the published artifacts and generating sourcemaps in `dist` makes debugging much nicer. Previously we generated the sourcemaps but didn't bundle the source so they pointed to files that didn't exist.